### PR TITLE
Add 'gladys start' scene trigger

### PR DESCRIPTION
**Pull request by callemand**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [x] If your changes affects code, did your write the tests?
- [x] Are tests passing? (`npm test` on both front/server)
- [x] Is the linter passing? (`npm run eslint` on both front/server)
- [x] Did you run prettier? (`npm run prettier` on both front/server)
- [x] If you are adding a new features/services, did you run integration comparator? (`npm run compare-translations` on front)
- [x] Did you test this pull request in real life? With real devices? If this development is a big feature or a new service, we recommend that you provide a Docker image to the community ([french forum](https://community.gladysassistant.com/)/[english forum](https://en-community.gladysassistant.com/)) for testing before merging.

### Description of change

Adding "gladys start" trigger into scene
Screenshot 2023-10-19 at 09 24 39
Screenshot 2023-10-19 at 09 24 15

### CODE REVIEW

File: front/src/routes/scene/edit-scene/triggers/GladysStartTrigger.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Component } from 'preact';
+import { connect } from 'unistore/preact';
+
+class SunriseSunsetTrigger extends Component {
+  render({}, {}) {
+    return (
+      <div>
+        <div class="row">
+          <div class="col-sm-12">COUCOU //TODO</div>
```
Comment (Pierre-Gilles) [inline front/src/routes/scene/edit-scene/triggers/GladysStartTrigger.jsx]: Forgot this? 😄 Maybe let's add a little sentence explaining what this trigger is doing?

Screenshot 2023-10-16 at 15 55 05

File: server/test/lib/scene/scene.checkTrigger.test.js [hunk 2]
```diff
@@ -446,6 +446,41 @@ describe('scene.checkTrigger', () => {
...
+            type: ACTIONS.LIGHT.TURN_OFF,
+            devices: ['light-1'],
+          },
+        ],
+      ],
+      triggers: [
+        {
+          type: EVENTS.SYSTEM.START,
+          area: 'area-1',
+          user: 'tony',
```
Comment (Pierre-Gilles) [inline server/test/lib/scene/scene.checkTrigger.test.js]: Those attributes are not linked to the trigger

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]

File: front/src/routes/scene/edit-scene/triggers/GladysStartTrigger.jsx (hunk 1 above)
Comment (callemand) [reply front/src/routes/scene/edit-scene/triggers/GladysStartTrigger.jsx]: Oops fixed now

File: server/test/lib/scene/scene.checkTrigger.test.js (hunk 2 above)
Comment (callemand) [reply server/test/lib/scene/scene.checkTrigger.test.js]: Oops bad copy/paste

File: front/src/config/i18n/fr.json [hunk 3]
```diff
@@ -1658,6 +1661,9 @@
         "minute": "minutes",
         "hour": "heures",
         "day": "jours"
+      },
+      "gladysStart": {
+        "description": "Cette scène sera executé a chaque démarage de Gladys"
```
Comment (Pierre-Gilles) [inline front/src/config/i18n/fr.json]: Typos: "Cette scène sera exécutée à chaque démarrage de Gladys"

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]
Little typos!

**Review by Pierre-Gilles — APPROVED** [review]
Good for me!

**Pull request merged**